Game/Source/ThirdPersonShooter/AIGym/ScaleTestableCharacter.cpp

```diff
@@ -133,7 +135,7 @@ void AScaleTestableCharacter::BeginPlay()
 {
 	Super::BeginPlay();
 
-	if (FParse::Param(FCommandLine::Get(), TEXT("simulatedplayer")))
+	if (FParse::Param(FCommandLine::Get(), TEXT("simulatedplayer")) || USimPlayerBPFunctionLibrary::IsSimulatedPlayer(this))
```

Comment:
nit: why doesn't the static function do the args checking?

Reply:
good point - it does do the command line arg check as well. Will remove the first clause